SpatialGDK/Documentation/content/porting-unreal-project-to-gdk.md

```diff
@@ -1,32 +1,47 @@
 <%(Callout type="warn" message="This [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS Unreal GDK is for evaluation and feedback purposes only, with limited documentation - see the guidance on [Recommended use]({{urlRoot}}/index#recommended-use)")%>
 
-# How to port a native Unreal project to the Unreal GDK
```

Comment:
I think we need to have the cloning info for the user's game project in here?

Reply:
What do you mean by the `cloning info` here?
The `## Prerequisites` specifies the user must have reached the cloning section from the set up guide.

Reply:
Ignore this!